influxql/point.go

```diff
@@ -152,3 +153,13 @@ func encodeTags(m map[string]string) []byte {
 	}
 	return b
 }
+
+// ParseTags returns an instance of Tags for a comma-delimited list of key/values.
+func ParseTags(s string) Tags {
```

Comment:
Is this used to parse our the tags from key stored on disk? If so, should probably use a func in models.Point because this is missing some escape char handling.